.vscode/settings.json

```
@@ -33,5 +33,6 @@
   ],
   "editor.codeActionsOnSave": {
     "source.fixAll.eslint": true
-  }
+  },
+  "editor.tabSize": 2
```

Use the `editorconfig` extension for vscode.

```suggestion
  }
```